garageScript/c0d3-app
# fix: Submission review count shows with none submissions

Closes #2208 

Merging this PR will stop the submissions review count from being displayed if there's no submissions.


Before:

<img width="877" alt="image" src="https://user-images.githubusercontent.com/35906419/189169111-dfc683e8-b12f-4f83-8b5e-350900014c3c.png">


After:

<img width="881" alt="image" src="https://user-images.githubusercontent.com/35906419/189168994-41712c1a-4f9e-44dd-8aff-2d1ba7414b8d.png">


To test it:
1. Go to `/curriculum`
2. Notice how it doesn't show for some of the lessons
